config/source/cli/cli.go

```diff
@@ -0,0 +1,138 @@
+// Package cli is the Cli component of orb.
+package cli
+
+import (
+	"errors"
+	"net/url"
+	"os"
+
+	"go-micro.dev/v5/codecs"
+	"go-micro.dev/v5/config/source"
+	"go-micro.dev/v5/log"
+	"go-micro.dev/v5/util/container"
+)
+
+// ParseFunc is the subplugin of source/cli.
+type ParseFunc func(config *Config, flags []*Flag, args []string) error
+
+func init() {
+	err := source.Plugins.Add(New())
+	if err != nil {
+		panic(err)
+	}
+}
+
+// Source is the cli source.
+type Source struct{}
+
+// New creates a new cli source.
+func New() source.Source {
+	return &Source{}
+}
+
+// Schemes returns the supported schemes by this plugin.
+func (s *Source) Schemes() []string {
+	return []string{"cli"}
+}
+
+// PrependSections indicates whether this needs sections to be prepended,
+// which is true in this case.
+func (s *Source) PrependSections() bool {
+	return true
+}
+
+// String returns the name of this plugin.
+func (s *Source) String() string {
+	return "cli"
+}
+
+// Read creates the subplugin for the given url,
+// creates its config after and then executes it.
+func (s *Source) Read(u *url.URL) source.Data {
+	result := source.Data{
+		Data: make(map[string]any),
+	}
+
+	pName := u.Host
+	if pName == "" {
+		pName = "urfave"
+	}
```

Comment:
If I understand correctly, this is setting the default cli plugin to urfave? 
We should make this a global like `DefeaultCLIPLugin` or smth.